tdrs-frontend/src/components/Profile/Profile.jsx

```diff
@@ -72,11 +72,7 @@ function Profile() {
               className="sign-in-button"
               type="button"
               id="loginDotGovSignIn"
-              onClick={(event) => {
-                event.preventDefault()
-                window.location.href =
-                  'https://idp.int.identitysandbox.gov/account'
-              }}
+              url={`${process.env.REACT_APP_LOGIN_GOV_URL}`}
```

Comment:
now grabbing this link from the environment variables. It will be set differently in dev/staging vs main.